bw2parameters/pint.py

```diff
@@ -1,36 +1,34 @@
+from pathlib import Path
 import re
-
 import pint.util
 from pint import DimensionalityError, Quantity, UndefinedUnitError, UnitRegistry
 from pint.util import string_preprocessor
 
+UNITS_FILE = Path(__file__).parent / "ecoinvent_units.txt"
```

Comment:
This name should reflect that it is ecoinvent-specific.

Reply:
Yeah, I was a bit sloppy here. Confession: not all the units in `ecoinvent_units.txt` are really from ecoinvent. I added some of my own. In general I think it's a good idea to have a place where the user can add custom definitions. I'm just not sure if we should have an extra file for them. What do you think?